### PR TITLE
fix: video status badge translation

### PR DESCRIPTION
**Pull request by CefBoud**
This PR is related to:
* https://github.com/openedx/edx-platform/pull/33939/

The changes are minimal:
* TODO Comments are removed. 
* The usage of the  `status` field is replaced by the new `status_nontranslated` field (introduced in the PR above).
* The display of a video's status in the status badge is done through `FormattedMessage` instead of a plain render to handle translation within React.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @CefBoud! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by mphilbrick211** [comment]
Hi @openedx/teaching-and-learning! This is ready for review. Thanks!

**Comment by openedx-webhooks** [comment]
I've created issue [TNL-11332](https://2u-internal.atlassian.net/browse/TNL-11332) in the private 2U Jira.

**Comment by tecoholic** [comment]
@CefBoud :+1: Just left a small nit and a question. 

- [x] I tested this: Used the instructions from https://github.com/openedx/frontend-lib-content-components/pull/414 and https://github.com/openedx/edx-platform/pull/33939 to verify that the filtering of videos works when the user's language is not English.
- [x] I read through the code
- [x] I checked for accessibility issues
- [ ] Includes documentation

File: src/editors/containers/VideoGallery/messages.js
Comment (CefBoud) [reply src/editors/containers/VideoGallery/messages.js:57]: It is used in the [status badge ](https://github.com/openedx/frontend-lib-content-components/pull/438/files#diff-2a9be15bb59a101e9361976e74115bea6eb9e470a74450b7cd119d1b0852d2a9R48)display.

Comment (tecoholic) [reply src/editors/containers/VideoGallery/messages.js:57]: Ah! Cool. :)

**Comment by mphilbrick211** [comment]
Hi @openedx/2u-tnl! Would someone be able to please review / merge this for us? If so, we can work on the branch conflicts. Thanks!

**Comment by mphilbrick211** [comment]
> Hi @openedx/2u-tnl! Would someone be able to please review / merge this for us? If so, we can work on the branch conflicts. Thanks!

Friendly ping on this, @openedx/2u-tnl!

**Comment by farhaanbukhsh** [comment]
@mphilbrick211  any updates here?

**Review by KristinAoki — APPROVED** [review]

**Comment by KristinAoki** [comment]
@CefBoud conflicts need to be resolved before this can be merged

**Comment by CefBoud** [comment]
@KristinAoki Thanks for the review. Fixed.

**Comment by openedx-webhooks** [comment]
@CefBoud 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-lib-content-components&entry.867055334=https://github.com/openedx/frontend-lib-content-components/pull/438&entry.1484655318=https://github.com/CefBoud&entry.752974735=2023-12-16+20:58&entry.1917517419=2024-05-13+19:26&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**
